src/Components/CreateImageWizard/utilities/useValidation.tsx

```diff
@@ -454,7 +454,7 @@ export function useDetailsValidation(): StepValidation {
   }, [blueprintId, name, setUniqueName, trigger, nameValid]);
 
   let nameError = '';
-  if (name && !nameValid) {
+  if (!nameValid) {
```

Comment:
We might want to add an empty value condition here so we're able to render a custom error message for a missing value.